modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddFilter.tsx

```diff
@@ -464,6 +464,7 @@ export function ModalAddFilter({
 							'there-are-no-columns-available'
 						)}
 						error={errors.selectedFilterBy}
+						id="modal-add-filter-by"
```

Comment:
Hi @Artartur. Please, rename the id to `modalAddFilterBy`.

Reply:
Done!